### PR TITLE
DS Import Warning Cleanup

### PR DESCRIPTION
**Pull request by k-milbury**
**DS-Task-import-warning-cleanup**
https://app.clickup.com/t/86dtt9dfa

**Changes**
Removed unused classes causing import error warnings on startup

**Testing**
Pull down the PR branch
Run `yarn install`
Run `yarn start` / `yarn develop`
Confirm that there are no more of these warnings
image
View the site, shouldn't be any styling/functionality changes

**Notes**
Looks way longer than it is, Prettier went to town on it lol

### CODE REVIEW

File: src/pages/cart.js [hunk 1]
```diff
@@ -36,99 +36,97 @@ export default function CartPage ({data}){
...
+            <div className={emptyStateContainer}>
+              <h1 className={title}>Your cart is empty</h1>
+              <p>
+                Looks like you haven’t found anything yet. We understand that
+                sometimes it’s hard to choose — maybe this helps:
+              </p>
+            </div>
+          ) : (
+            <>
+              <h1 className={title}>Your cart</h1>
```
Comment (k-milbury) [inline src/pages/cart.js:50]: just want to confirm on this one that it is fine, what I removed from the previous one didn't exist anyways but want to make sure that nothing else needs to go in there for that to work where it was a joint class name before
Previous:
` <h1 className={[emptyStateHeading, title].join(" ")}>`

Comment (klancashire) [reply src/pages/cart.js:50]: Yeah this is 100% okay, thank you!

**Review by klancashire — COMMENTED** [review]
Still getting a list of unused styles unfortunately when I run build: 

warn ./src/components/_homepage/hero.js
Attempted import error: 'dotGroup' is not exported from './hero.module.scss' (imported as 'sty').
warn ./src/components/_homepage/hero.js
Attempted import error: 'title' is not exported from './hero.module.scss' (imported as 'sty').
warn ./src/components/_homepage/instagram-feed.js
Attempted import error: 'feed' is not exported from './instagram-feed.module.scss' (imported as 'sty').
warn ./src/components/_homepage/testimonials.js
Attempted import error: 'dot' is not exported from './testimonials.module.scss' (imported as 'sty').
warn ./src/components/menus/TopMenu.js
Attempted import error: 'navClosed' is not exported from './TopMenu.module.scss' (imported as 'sty').
warn ./src/components/menus/TopMenu.js
Attempted import error: 'navClosed' is not exported from './TopMenu.module.scss' (imported as 'sty').
warn ./src/components/product-card.js
Attempted import error: 'ProductCard' is not exported from './product-card.module.scss' (imported as 'sty').
warn ./src/components/slices/awards-gallery.js
Attempted import error: 'award' is not exported from './awards-gallery.module.scss' (imported as 'sty').
warn ./src/components/slices/press-list.js
Attempted import error: 'PressList' is not exported from './press-list.module.scss' (imported as 'sty').
warn ./src/pages/cart.js
Attempted import error: 'emptyStateHeading' is not exported from './cart.module.scss' (imported as
'emptyStateHeading').

**Comment by klancashire** [comment]
Update: did a pull and all these errors have been resolved, merge away!

**Pull request merged**